workspace/resource_workspace_conf.go

```diff
@@ -138,7 +142,17 @@ func ResourceWorkspaceConf() common.Resource {
 			if err != nil {
 				return err
 			}
-			return w.WorkspaceConf.SetStatus(ctx, patch)
+			err = w.WorkspaceConf.SetStatus(ctx, patch)
+			// Tolerate errors like the following on deletion:
+			// cannot delete workspace conf: Some values are not allowed: {"enableGp3":"","enableIpAccessLists":""}
+			// The API for workspace conf is quite limited and doesn't support a generic "reset to original state"
+			// operation. So if our attempted reset fails, don't fail resource deletion.
+			var apiErr *apierr.APIError
+			if errors.As(err, &apiErr) && strings.HasPrefix(apiErr.Message, "Some values are not allowed") {
+				tflog.Warn(ctx, fmt.Sprintf("Failed to delete databricks_workspace_conf: %s. The workspace configuration may not be fully restored to its original state. For more information, see %s", apiErr.Message, docs.ResourceDocumentationUrl("workspace_conf")))
+				return nil
+			}
+			return err
```

Comment:
Do you know if this could lead to partial update, or if the conf API rejects everything if the payload contains at least 1 invalid setting?

Reply:
I have the same question - if everything is rejected, should we set each setting separately?

Reply:
IIUC, you're asking if partial updates succeed for most settings but fail for a few, we could restore most settings to their original state and minimize manual work for customers. If the API partially updates settings but fails for others, then I think we will get this desired behavior. Otherwise, the error message will include the settings that the customer needs to manually check. I will double check with the backing team to understand what might happen with this.